Gem/Code/Source/Components/Multiplayer/EnergyBallComponent.cpp

```diff
@@ -198,7 +198,6 @@ namespace MultiplayerSample
                     // Look for physics rigid body component and make impact updates
                     if (Multiplayer::NetworkRigidBodyComponent* rigidBodyComponent = handle.GetEntity()->FindComponent<Multiplayer::NetworkRigidBodyComponent>())
                     {
-                        const AZ::Vector3 explosionCentre = position;
```

Comment:
Alright who wrote "Centre"?

Reply:
The same person that probably has text "Colouring" for their IDE